airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingConsumerFactory.java

```diff
@@ -45,6 +46,7 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(SnowflakeInternalStagingConsumerFactory.class);
 
   private static final long MAX_BATCH_SIZE_BYTES = 1024 * 1024 * 1024 / 4; // 256mb
+  private static final String currentSyncPath = UUID.randomUUID().toString();
```

Comment:
Please, use java convention for constants. Uppercase with underscores.

Reply:
> Please, use java convention for constants. Uppercase with underscores.

renamed